webpack/assets/javascripts/react_app/common/I18n.js

```diff
@@ -4,9 +4,8 @@ import { addLocaleData } from 'react-intl';
 class IntlLoader {
   constructor(locale, timezone) {
     this.fallbackIntl = !global.Intl;
-
     [this.locale] = locale.split('-');
-    this.timezone = this.fallbackIntl ? 'UTC' : timezone;
+    this.timezone = this.fallbackIntl || timezone ? (timezone ? timezone : 'UTC') : global.Intl.DateTimeFormat().resolvedOptions().timeZone;
```

Comment:
The second timezone condition will be always true, right? And could we please use parenthesis?
```suggestion
    this.timezone = this.fallbackIntl ? 'UTC' : (timezone || global.Intl.DateTimeFormat().resolvedOptions().timeZone);
```

Reply:
Lint was complaining about the ternary..updated with if-else. I am not sure how to set the cookie with the frontend timezone and if we should update the backend timezone with that..

Reply:
Yeah, it's probably more readable. I've responded to the cookie in the comment as it is the main issue we need to solve for this one (we need backend and frontend aligned). I as a user wouldn't mind that much I have all the datetimes in different timezone, than if I have some dates in one timezone and one in another one.